crates/teloxide-core/src/types/message.rs

```diff
@@ -671,10 +669,7 @@ mod getters {
         /// Returns the user who sent the message.
         #[must_use]
         pub fn from(&self) -> Option<&User> {
```

Comment:
Could you deprecate `from` (the function)? If it's just a field access I don't think we need it. Also the same for `sender_chat`.

(don't forget to update the changelog too)

Reply:
I think you should remove it, I don't think `filter_from` makes much sense..

Reply:
@tar-xzf sorry, after re-reading this now I changed my mind >_>

`filter_from` still makes sense, given that it's an `Option`.

Please add it back. An `#[allow(deprecated)]` with a `// FIXME: change macro so that we can filter things without getters` should be fine.